src/common/utility.ts

```diff
@@ -403,7 +403,8 @@ export const updateURLAsPerUserLanguage = () => {
     const paths = current_path.split('/')
     const first_path = paths[1]
     const has_language_in_url = first_path in language_config
-    has_language_in_url && Cookies.set('user_language', first_path)
+    document.cookie =
+        has_language_in_url && 'user_language=' + first_path + ';domain=.' + getDomain() + ';path=/'
     const user_language = Cookies.get('user_language') || 'en'
```

Comment:
In case when we are not set Cookie, we have to rewrite it on:
```suggestion
    const user_language = document.cookie || 'en'
```